lib/orogen/gen/tasks.rb

```diff
@@ -306,6 +306,48 @@ def register_for_generation
             # <tt>_time</tt> to be added to the generated class (more specifically,
             # to the +Base+ subclass).
             module TaskContextGeneration
+                def initialize(project, name = nil, *args, **options)
+                    RTT_CPP.verify_valid_identifier(name, "task")
+                    super
+                end
+
+                def output_port(name, type, *args, **options)
+                    RTT_CPP.verify_valid_identifier(name, "output port")
+                    type = validate_interface_type(type)
+
+                    if type.name == "/std/vector<double>"
+                        Spec.warn(
+                            "#{type.name} is used as the port type for #{name}, "\
+                            "logging it will not be possible"
+                        )
+                    end
```

Comment:
what is the problem with logging a Vectord ?

Reply:
It is very historical - 10 yrs ago RTT had hardcoded type information for vector<double> *and* no way to overload that. It's not relevant anymore. But then, you should really not have a simple vector<double> on a port :P